tests/CMakeLists.txt

```diff
@@ -15,6 +15,11 @@ endmacro()
 if (PY_PYTEST)
     ssg_python_unit_tests("build-scripts")
     ssg_python_unit_tests("utils")
+    # The SSG unit test is special - the ssg subdir can't be in the PYTHONPATH.
+    add_test(
+        NAME "python-unit-ssg"
+        COMMAND "${PYTHON_EXECUTABLE}" -m pytest ${PYTEST_COVERAGE_OPTIONS} "${CMAKE_SOURCE_DIR}/tests/unit/ssg"
```

Comment:
could we rename it to ssg_module? sounds easier and will bring way less "wtf" moments in the future.

Reply:
Good suggestion, fixed in the rebase.

Reply:
Was this done or not? The hunk doesn't seem to have it.